spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe SearchController, :type => :controller do
+  render_views
+
+  describe "GET fees" do
+    before(:each) { get :fees }
+
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should contain the search form" do
+      expect(response.body).to include 'Fee search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.